tests/kafkatest/services/kafka/quorum_utils.py

```diff
@@ -0,0 +1,55 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# the types of metadata quorums we support
+zk_quorum = 'ZK' # ZooKeeper, used through the KIP-500 bridge release
+inproc_raft_quorum = 'INPROC_RAFT' # co-located KIP-500 Controllers, used during/after the KIP-500 bridge release
+remote_raft_quorum = 'REMOTE_RAFT' # separate KIP-500 Controllers, used during/after the KIP-500 bridge release
+
+# How we will parameterize tests that exercise all quorum styles
+all_quorum_styles = [zk_quorum, remote_raft_quorum, inproc_raft_quorum]
+# How we will parameterize tests that exercise all Raft quorum styles
+all_raft_quorums = [remote_raft_quorum, inproc_raft_quorum]
+# How we will parameterize tests that are unrelated to upgrades
+non_upgrade_quorums = [zk_quorum, remote_raft_quorum]
```

Comment:
I don't get the distinction between upgrade / non-upgrade.  That's not really related to quorum type, right?

Seems like we could just have all_separate to express "raft + zk" and all to express "raft + zk + raft_colocated"

Reply:
I eliminated this.  We can put it back in when we support bridge release upgrades and need to write the system tests.